.github/workflows/birdbox.yml

```diff
@@ -15,6 +15,11 @@ on:
       - '.github/workflows/birdbox.yml'
     branches:
       - 'master'
+  # TODO: REMOVEME. Temporary, to validate CI.
```

Comment:
Do you still need it?

Reply:
Nope, removing it just before submission. It did it's job: green checks for review :)

Reply:
Though perhaps we could leave it in, to avoid silently broken tests in the future. Next time the release tests break...